orca-sql/src/main/kotlin/com/netflix/spinnaker/config/SqlConfiguration.kt

```diff
@@ -161,4 +170,13 @@ class SqlConfiguration {
   fun lockProvider(datasource: DataSource): LockProvider {
     return JdbcTemplateLockProvider(datasource)
   }
+
+  @Bean
+  @ConditionalOnProperty("execution-repository.sql.pipeline-ref.enabled")
+  fun pipelineRefTriggerDeserializer(@Qualifier("mapper") mapper: ObjectMapper): CustomTriggerDeserializerSupplier {
+    val customTrigger = PipelineRefTriggerDeserializerSupplier()
+    TriggerDeserializer.customTriggerSuppliers.add(customTrigger)
```

Comment:
This is required because CustomTriggerSupplier is a list in a companion block. Everything it is initialized with empty and this ensure Orca will know how to deserialize the PipelineRefTrigger